lib/puppetserver/ca/puppet_config.rb

```diff
@@ -143,6 +158,10 @@ def parse_text(text)
       def explicitly_given_config_file_or_default_config_exists?
         !@using_default_location || File.exist?(@config_path)
       end
+
+      def run(command)
+        %x'#{command}'
```

Comment:
What's %x do?

Reply:
It's like backticks, shells out and returns the commands output

Reply:
but is a lot more grep-able than backticks

Reply:
I think they handle errors slightly differently, I should maybe double check that.

Reply:
From what I've found they do handle errors the same.